service/src/main/java/com/myhome/services/springdatajpa/SecurityTokenSDJpaService.java

```diff
@@ -0,0 +1,45 @@
+package com.myhome.services.springdatajpa;
+
+import com.myhome.domain.MyHomeTokenType;
+import com.myhome.domain.SecurityToken;
+import com.myhome.repositories.SecurityTokenRepository;
+import com.myhome.services.SecurityTokenService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class SecurityTokenSDJpaService implements SecurityTokenService {
+
+  private final SecurityTokenRepository securityTokenRepository;
+
+  @Value("${users.passResetTokenTime}")
+  private int passResetTokenTime;
```

Comment:
Let's change it to `Duration` instead of `int`.
https://stackoverflow.com/questions/51818137/spring-boot-2-converting-duration-java-8-application-properties